dev-db/etcd-wrapped/files/etcd-wrapped.ebuild

```diff
@@ -0,0 +1,97 @@
+#
+# Copyright (c) 2016 CoreOS, Inc.. All rights reserved.
+# Distributed under the terms of the GNU General Public License v2
+# $Header:$
+#
+
+EAPI=6
+
+inherit systemd
+
+DESCRIPTION="etcd Wrapper"
+HOMEPAGE="https://github.com/coreos/etcd"
+KEYWORDS="amd64 arm64"
+
+LICENSE="Apache-2.0"
+IUSE="etcdctl"
+
+VERSION=${PV}
+VERSION=${VERSION/_alpha/-alpha.}
+VERSION=${VERSION/_beta/-beta.}
+
+case ${VERSION} in
+0*)
+	SLOT="0"
+	;;
+2*)
+	SLOT="2"
+	;;
+3*)
+	SLOT="3"
+	;;
+*)
+	die "Unknown etcd version ${VERSION}"
+	;;
+esac
+
+
+SLOT_MANGLED=${SLOT/0/}
+IMG_FILENAME="etcd-v${VERSION}-linux-${ARCH}.aci"
```

Comment:
What exactly do you mean by guard? We need to specify the ARCH in the SRC_URI because we need the appropriate container image.